### PR TITLE
change Log to log.L()

### PR DESCRIPTION
**Pull request by basit9958**
Signed-off-by: Basit Hasan <hasanbasit13@gmail.com>

### What problem does this PR solve?
fixex #2878 

### Related changes

- [ ] Need to update `chaos-mesh/website`
- [ ] Need to update `Dashboard UI`
- Need to **cheery-pick to release branches**
  - [ ] release-2.1
  - [ ] release-2.0

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] No code
- [ ] Manual test (add steps below)

Side effects

- [ ] Breaking backward compatibility

### Release note 

```text
Please add a release note.

You can safely ignore this section if you don't think this PR needs a release note.
```

### DCO

If you find the DCO check fails, please run commands like below (Depends on the actual situations. For example, if the failed commit isn't the most recent) to fix it:

```shell
git commit --amend --signoff
git push --force
```

### CODE REVIEW

**Review by Andrewmatilde — APPROVED** [review]
LGTM

File: pkg/dashboard/apiserver/utils/auth.go [hunk 1]
```diff
@@ -48,7 +47,7 @@ func AuthMiddleware(c *gin.Context, config *config.ChaosDashboardConfig) {
 	if ns == "" && !config.ClusterScoped && config.TargetNamespace != "" {
 		ns = config.TargetNamespace
 
-		log.V(1).Info("Replace query namespace with", ns)
+		log.L().WithName("auth middleware").V(1).Info("Replace query namespace with", ns)
```
Comment (STRRL) [inline pkg/dashboard/apiserver/utils/auth.go:50]: please use `kebab-case` for logger name

File: pkg/dashboard/apiserver/utils/error.go [hunk 2]
```diff
@@ -67,7 +68,7 @@ func SetAPIError(c *gin.Context, err *errorx.Error) {
 		FullText: fmt.Sprintf("%+v", err),
 	}
 
-	Log.Error(err.Cause(), typeName)
+	log.L().Error(err.Cause(), typeName)
```
Comment (STRRL) [inline pkg/dashboard/apiserver/utils/error.go:71]: please also assign logger name here.

**Review by STRRL — COMMENTED** [review]
rest LGTM! 

PTAL @basit9958

**Comment by basit9958** [comment]
I did git pull and pushed  now I also see merge branch commit 3f2af205d8d6d0d0bad0433aaac196b4a0fd59f7  here should I revert this @STRRL

File: pkg/dashboard/apiserver/utils/error.go [hunk 3]
```diff
@@ -67,7 +68,7 @@ func SetAPIError(c *gin.Context, err *errorx.Error) {
 		FullText: fmt.Sprintf("%+v", err),
 	}
 
-	Log.Error(err.Cause(), typeName)
+	log.L().WithName("kebab-case").Error(err.Cause(), typeName)
```
Comment (Andrewmatilde) [inline pkg/dashboard/apiserver/utils/error.go:71]: PTAL https://www.theserverside.com/definition/Kebab-case
I think he means auth-middleware.

File: pkg/dashboard/apiserver/utils/auth.go [hunk 4]
```diff
@@ -48,7 +47,7 @@ func AuthMiddleware(c *gin.Context, config *config.ChaosDashboardConfig) {
 	if ns == "" && !config.ClusterScoped && config.TargetNamespace != "" {
 		ns = config.TargetNamespace
 
-		log.V(1).Info("Replace query namespace with", ns)
+		log.L().WithName("kebab-case").V(1).Info("Replace query namespace with", ns)
```
Comment (Andrewmatilde) [inline pkg/dashboard/apiserver/utils/auth.go:50]: ditto

Comment (basit9958) [reply pkg/dashboard/apiserver/utils/auth.go:50]: Okay

**Comment by STRRL** [comment]
Hi @basit9958 please run `make check` on you local machine then commit the changes.

**Review by STRRL — APPROVED** [review]

**Pull request merged**